Assets/Mirror/Runtime/NetworkTime.cs

```diff
@@ -73,14 +73,17 @@ static NetworkTime()
         // TODO does this need to be public? user should only need NetworkTime.time
         public static double rttStandardDeviation => Math.Sqrt(rttVariance);
 
+        [UnityEngine.RuntimeInitializeOnLoadMethod]
         public static void Reset()
```

Comment:
regarding my above comment: this is what I meant. we already have Reset().
might as well call them all Reset().

Reply:
Reset is a special unity method, same like Start and Awake.  So I am not sure if we want all those methods be called Reset or maybe we call them ResetStatics/ResetState or something similar

Reply:
Added comment above it

Reply:
it's called ResetStatics now